kernel/Kconfig

```diff
@@ -928,4 +928,16 @@ config THREAD_LOCAL_STORAGE
 
 endmenu
 
+menu "Device Options"
+
+config HAS_DYNAMIC_DEVICE_HANDLES
+	bool
+	depends on (PM_DEVICE_POWER_DOMAIN_DYNAMIC_NUM > 0)
+	default y
+	help
+	  Hidden option to enable support for dynamically add devices
+	  to supported handles.
```

Comment:
```suggestion
	  Hidden option that makes device handles non-const. It can be useful when device handles have to be manipulated at runtime.
```

Reply:
`Hidden option that makes possible to manipulate device handles at  runtime.`

I think the non-const is not necessary, that is not the only thing that this option does, so instead of partially describe the implementation I rather focus in the behavior / feature.